ocs_ci/utility/utils.py

```diff
@@ -496,25 +496,32 @@ def run_cmd(
     return mask_secrets(completed_process.stdout.decode(), secrets)
 
 
-def run_cmd_interactive(cmd, prompts_answers, timeout=300):
+def run_cmd_interactive(
+    cmd, prompts_answers, timeout=300, string_answer=False, raise_exception=True
+):
     """
     Handle interactive prompts with answers during subctl command
 
     Args:
         cmd(str): Command to be executed
         prompts_answers(dict): Prompts as keys and answers as values
         timeout(int): Timeout in seconds, for pexpect to wait for prompt
-
+        string_answer (bool): string answer
+        raise_exception (bool): raise excption
     Raises:
         InteractivePromptException: in case something goes wrong
 
     """
     child = pexpect.spawn(cmd)
     for prompt, answer in prompts_answers.items():
```

Comment:
can we add a condition when there are no prompts?

Reply:
if there is no prompt we don need to use this funcion